test/packages/file_upload_example/file_upload/files/elasticsearch/ilm/files.json

```diff
@@ -0,0 +1,15 @@
+{
+    "policy": {
+        "phases": {
+            "hot": {
+                "min_age": "0ms",
+                "actions": {
+                    "rollover": {
+                        "max_size": "10gb",
+                        "max_age": "1d"
+                    }
+                }
+            }
+        }
+    }
+}
```

Comment:
Do we expect ILM policies for this feature to be different between packages?

Reply:
I could only see ILM policies being different if Fleet and Security want to delete docs at different rates.

@pzl @kpollich @joshdover @juliaElastic - do you see any reason for us to have different ILM policies?  On the Endpoint side, we were considering fairly quick cleanup, perhaps removing files that are more than 1-3 days old.  Is that OK with Fleet?

Reply:
I think it'd be fine to globally apply the same ILM policy for all file upload indices - Fleet doesn't have any unique constraints here that I'm aware of.